### PR TITLE
doc: Add portable pdbs to symbol server compatibility

### PR DESCRIPTION
**Pull request by loewenheim**
#skip-changelog

### CODE REVIEW

**Review by ashwoods — APPROVED** [review]

**Review by Swatinem — COMMENTED** [review]
Portable PDB should mention somewhere that the `<Age>` is masked as all-`F`.

**Comment by loewenheim** [comment]
Oh, and we should probably also mention that `nuget` requires that checksum header for ppdbs. Where would be a good place to put that?

**Comment by Swatinem** [comment]
Can we link to some upstream docs for that? I don’t quite remember if that fact is even documented anywhere properly…

**Comment by loewenheim** [comment]
There is documentation for the nuget symbol server [here](https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg#nugetorg-symbol-server), but it doesn't say anything about checksums.

@mattjohnsonpint, do you happen to know if there's anything we can link to?

**Comment by mattjohnsonpint** [comment]
As far as I can tell, it's undocumented.  The best writeup on it I can find is our own.
https://github.com/getsentry/rfcs/blob/main/text/0013-portable-pdb.md#nuget-symbol-server-lookups

The only thing from Microsoft is this issue and its corresponding PR.
https://github.com/dotnet/symstore/issues/102

**Review by Swatinem — APPROVED** [review]

File: docs/advanced/symbol-server-compatibility.md [hunk 1]
```diff
@@ -312,6 +329,7 @@ The debug id is in all cases lowercase in hex format and computed as follows:
 
 - **PE**: `<Signature><Age>` (age in hex, not padded)
 - **PDB**: `<Signature><Age>` (age in hex, not padded)
+- **Portable PDB**: `<Signature>FFFFFFFF`
```
Comment (Swatinem) [inline docs/advanced/symbol-server-compatibility.md:332]: I believe for the unified layout we actually use the full debug id, can you double-check? It should be consistent what symsorter would do in that case, which might also be a good idea to check

**Review by Swatinem — APPROVED** [review]

**Pull request merged**